blockbase/package.json

```diff
@@ -3,37 +3,26 @@
   "version": "2.0.9",
   "description": "Blank Parent Theme",
   "bugs": {
-    "url": "https://github.com/Automattic/themes/issues"
+    "url": "https://github.com/Automattic/themes/issues/new?label=[Theme]%20Blockbase"
```

Comment:
I like this, should we change it on the rest of the themes?

Reply:
Yes, wait for my next PR!